about.html

```diff
@@ -0,0 +1,162 @@
+            type="button"
+            class="hamburger-btn"
+            aria-expanded="false"
+          ></button>
+          <ul data-visible="false" class="header-ul">
+            <li class="header-item bot">
+              <a class="red" href="index.html">home</a>
+            </li>
+            <li class="header-item bot">
+              <a class="li-btn" href="index.html#program">Program</a>
+            </li>
+            <li class="header-item bot">
+              <a class="li-btn" href="index.html#join">Join</a>
+            </li>
+            <li class="header-item bot">
+              <a class="li-btn" href="index.html#partner">Sponsors</a>
+            </li>
+            <li class="header-item bot"><a class="li-btn" href="#">News</a></li>
+            <li class="header-item square">
+              <a class="li-btn" href="#">CC Campaign</a>
+            </li>
+          </ul>
+        </header>
+      </nav>
+    </main>
+    <section class="presentation">
+      <h2 class="greeting">"Hello! sharing world"</h2>
+      <h1>
+        AGILITY CONTEST<br />DOG TRAINERS
+      </h1>
+      <p class="present-desc">
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Harum, qui?
+        Minima necessitatibus ex eius, culpa excepturi aliquid, temporibus
+        reprehenderit cupiditate alias, ullam quos. Totam ipsam incidunt odio
+        tempore ratione ea?
+      </p>
+      <p class="contact">
+        please contact us per Email for any further questions about CC Global
+        summit 2015!
+      </p>
+      <a href="#" class="mail">
+        thisrandomemail@gmail.com <i class="bx bx-chevron-right"></i>
+      </a>
+    </section>
+    <section class="logo-cont">
+      <h2 class="logo-title">this is the logo presentation</h2>
+      <div class="line"></div>
+      <p class="logo-desc">
+        Lorem ipsum dolor sit amet consectetur adipisicing elit.
+      </p>
+      <a class="logo logo-show">
+        <img alt="Agility contest" class="logo-img-show" src="./assets/images/detailed-logo.jpg" />
+        <h3 class="logo-text">
+          agility <br />
+          contest
+        </h3>
+      </a>
+    </section>
+    <section class="past-events">
+      <h2 class="events-title">See the pase agility contests</h2>
+      <div class="line"></div>
+      <p class="event-subtitle">
+        Take a look at the last two Agility contests which took place in Buenos
+        Aires and in Warsaw
+      </p>
+      <div class="events-pics">
+        <div class="event event1">
+          <div class="bg-event">
+            <p class="event-year">2015</p>
+            <p class="event-detail">Agility contest 2013 in Buenos Aires</p>
+          </div>
+        </div>
+        <div class="event event2">
+          <div class="bg-event">
+            <p class="event-year">2015</p>
+            <p class="event-detail">Agility contest 2013 in Buenos Aires</p>
+          </div>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] On mobile view the second image is overlapping with the partner's section. Please try and fix it.